wasm/typescript/test/browser/demo-app/write-parameter-files-load-sample-inputs.ts

```diff
@@ -1,30 +1,49 @@
-export default async function writeParameterFilesLoadSampleInputs (model, preRun=false) {
-  const dataUrl = 'https://w3s.link/ipfs/bafybeifq7m3mb4m3mgbhft2vqejjgwxx4azmmzuxwf5cubajpyaf4hq2gq/data/input'
+export default async function writeParameterFilesLoadSampleInputs(
+  model,
+  preRun = false
```

Comment:
For now it's OK to me, but maybe another improvement could be to specify the _type_ of these function parameters. Especially because that's what TypeScript is advertising to be, at https://www.typescriptlang.org/:  _a strongly typed programming language_. Right?

Anyway, I think our first priority is to get this PR merged 🚀 !